### PR TITLE
N21-1336  FEATURE_COMPUTE_TOOL_STATUS_WITHOUT_VERSIONS_ENABLED 

### PR DESCRIPTION
**Pull request by IgorCapCoder**
add feature flag and set it to true as default.
This is a temporary flag as a failsafe to hide a new way of determining a tool status. Instead of using versions, we are going to validate the paramters on the respective scope. This will allow us to determine the outdated scope of a ctl tool more easily, which will be then reflected in more precise instructions in error messages.

# Description

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/N21-1336
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/4534

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Comment by Loki-Afro** [comment]
very descriptive

File: ansible/group_vars/reference/cfg.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ ES_DOMAIN: https://repository.sc-staging-edusharing.dbildungscloud.org
 H5P_FRAME_SRC_URLS: "https://rendering.services.sc-staging-edusharing.dbildungscloud.org/"
 
 BETTERMARKS_CSP_URL: "https://apps-ci00.bettermarks.com"
+
+FEATURE_COMPUTE_TOOL_STATUS_WITHOUT_VERSIONS_ENABLED: "false"
```
Comment (MarvinOehlerkingCap) [inline ansible/group_vars/reference/cfg.yml:11]: why is it enabled on dev and prod but disabled on ref? just set the feature to true in the config.yml

**Review by MarvinOehlerkingCap — CHANGES_REQUESTED** [review]

File: ansible/group_vars/reference/cfg.yml (hunk 1 above)
Comment (IgorCapCoder) [reply ansible/group_vars/reference/cfg.yml:11]: fixed :)

**Comment by IgorCapCoder** [comment]
> very descriptive

Added words ;)

**Review by MarvinOehlerkingCap — APPROVED** [review]

**Pull request merged**